mongodb/pom.xml

```diff
@@ -12,6 +12,11 @@
     <name>Hibernate OGM for MongoDB</name>
     <description>Persist objects in MongoDB</description>
 
+    <properties>
+        <mongodbProvider>mongodb</mongodbProvider>
+        <slf4jVersion>1.7.10</slf4jVersion>
```

Comment:
Out of interest, we do you override the version? The one from the BOM seems to work fine.